org.geogebra.GeoGebra.json

```diff
@@ -0,0 +1,44 @@
+{
+    "app-id": "org.geogebra.GeoGebra",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "geogebra",
+    "finish-args": [
+        "--share=ipc",
+        "--share=network",
+        "--socket=x11",
+        "--socket=wayland",
+        "--filesystem=home"
+    ],
+    "modules": [
+        {
+            "name": "geogebra",
+            "buildsystem": "simple",
+            "sources": [
+                {
+                    "type": "archive",
+                    "url": "https://download.geogebra.org/installers/5.0/GeoGebra-Linux-Portable-5-0-367-0.tar.bz2",
```

Comment:
Looks like this bundles the entire JRE for both 32 and 64bit. At the very least this should only install the one used.